src/ecfeed.hpp

```diff
@@ -0,0 +1,1064 @@
+#pragma once
+#include "picojson.h"
```

Comment:
We should include picojson code here directly and add a separate license and mention it in the copyright notice somehow. We don't want the user to need to download it separately. Everything needed for ecfeed to work should be in this single file.

Reply:
Done.